.evg.yml

```diff
@@ -74,6 +74,37 @@ buildvariants:
 tasks:
   - name: "build"
     commands:
+      # fetch mongosqltranslate libraries for each platform before building
+      # linux arm
+      - func: "fetch libmongosqltranslate"
+        vars:
+          arch: "arm"
+          platform: "linux"
+          lib_prefix: "lib"
+          ext: "so"
+      # linux x86
+      - func: "fetch libmongosqltranslate"
+        vars:
+          arch: "x86_64"
+          platform: "linux"
+          lib_prefix: "lib"
+          ext: "so"
+      # macos arm
+      - func: "fetch libmongosqltranslate"
+        vars:
+          arch: "arm"
+          platform: "macos"
+          lib_prefix: "lib"
+          ext: "dylib"
+      # macos arm
+      - func: "fetch libmongosqltranslate"
+        vars:
+          arch: "x86_64"
+          platform: "macos"
+          lib_prefix: "lib"
```

Comment:
Just to be clear, when we eventually reuse the `"fetch libmongosqltranslate"` function for Windows, this argument will make more sense. Right now, every invocation of the function provides the same value for this argument. But the Windows library will _not_ start with "lib", so when we refactor to use this function for Windows we'll omit this argument for that invocation.

Reply:
Field [SQL-2406](https://jira.mongodb.org/browse/SQL-2406) for this.